### ISSUE TITLE
Scoreboards

### ISSUE THREAD

**Issue by gratimax** [SpongePowered/SpongeAPI#293]
The Sponge API needs scoreboards, see the [wiki article](http://minecraft.gamepedia.com/Scoreboard) for how they work. We're interested in scoreboard API proposals, mostly.

I know @Aaron1011 has been working on this, waiting on him for a PR/wip PR.

**Comment by gratimax** [comment]
Closing in favor of #299
